CODEOWNERS

```diff
@@ -2,4 +2,5 @@
 # for code within the repository
 # https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
 
+/pkgs/cli_config/ @dcharkes @HosseinYousefi
```

Comment:
Having an entry here is your choice; we don't generally use the CODEOWNERS file. Elias wanted to use one (or experiment with one?).

The benefit here is that you'll automatically be put as a reviewer for every PR that touches a file matching the glob above.

Reply:
Check, removed again.